tests/chia_log/handlers/test_wallet_added_coin_handler.py

```diff
@@ -3,24 +3,28 @@
 from pathlib import Path
 import copy
 
+# lib
+import confuse
+
 # project
 from src.chia_log.handlers.wallet_added_coin_handler import WalletAddedCoinHandler
 from src.notifier import EventType, EventService, EventPriority
-from src.config import Config
 
 
 class TestWalledAddedCoinHandler(unittest.TestCase):
     def setUp(self) -> None:
         config_dir = Path(__file__).resolve().parents[3]
-        config = Config(config_dir / "config-example.yaml")
-        self.handler_config = config.get_handlers_config()[WalletAddedCoinHandler.config_name()]
+        config = confuse.Configuration("chiadog", __name__)
+        config.set_file(config_dir / "src/default_config.yaml")
+        config.set_file(config_dir / "config-example.yaml")
+        self.handler_config = config["handlers"][WalletAddedCoinHandler.config_name()]
```

Comment:
Tied to another comment, maybe the handler class can have a method that returns the fully materialized view from the rootview. Then the code here would be slightly cleaner:
```
self.config_view_handler = WalletAddedCoinHandler.config_view(root_view)
```